MathlibTest/CountHeartbeats.lean

```diff
@@ -1,4 +1,4 @@
-import Mathlib.Tactic.Ring
+import Mathlib.Util.CountHeartbeats
```

Comment:
Thanks for reducing the imports!  I was very unhappy about the earlier dependencies.